searchbase/lib/src/searchStateController.dart

```diff
@@ -0,0 +1,336 @@
+import 'package:searchbase/searchbase.dart';
+import 'types.dart';
+import 'searchcontroller.dart';
+import 'constants.dart';
+
+/// It stores the state of the SearchController at any time.
+class SearchControllerState {
+  final Aggregations? aggregationData;
```

Comment:
Docs are missing for all the properties.

Reply:
Fixed here 👉  https://github.com/appbaseio/flutter-searchbox/pull/21/commits/e43591b8d06971646f10850237673cc68e5b4e9c